ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -337,18 +337,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
-- as: azure-ipi-private-p2-f14
-  cron: 34 16 12,27 * *
+- as: azure-ipi-private-workers-marketimage-p2-f14
+  cron: 34 16 1,10 * *
```

Comment:
You generated the cron setting by yourself? Not from `generate-cron-entry.sh` tools?
```
[jianlinliu@jialiu-laptop openshift-tests-private]$ generate-cron-entry.sh azure-ipi-private-workers-marketimage-p2-f14 openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml
cron: 55 13 2,17 * *
```

Reply:
No, I manually updated it

Reply:
all the cron setting is supposed to be generated from `generate-cron-entry.sh` tools, but not generate it manually.

Reply:
btw, For nightly job, I suggest to keep the same frequency as before on `f14`

Reply:
Done